app/main/posts/common/post-actions.directive.js

```diff
@@ -47,7 +49,10 @@ function PostActionsDirective(
         }
 
         function deletePost() {
+            LoadingProgress.setLoadingState({isLoading: true});
```

Comment:
Sometimes the loading-state needs to be set manually since no transitions are made, thats why I am setting it sometimes and sometimes not.

Reply:
why are we using the full state object instead of sending a bool, or even having a setLoadingStateOn/setLoadingStateOff ? I can't remember if there was a reason or just preference

Reply:
Its because we need to separate loading and saving, we don't want to make the save-button say 'saving' as soon as something is loading, only when it is actually loading and I tried to be clever and put them at the same place... not sure if thats the best way though...